### PR TITLE
GHA: add color output

### PR DESCRIPTION
**Pull request by mathause**
Should add color to the output on GHA

### CODE REVIEW

**Comment by veni-vidi-vici-dormivi** [comment]
Uhh that's cool thank you!

**Review by veni-vidi-vici-dormivi — APPROVED** [review]
Why three? I can't really find documentation about this?

**Comment by mathause** [comment]
No idea :man_shrugging: - I picked it up from xarray

File: .github/workflows/ci-workflow.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  FORCE_COLOR: 3
```
Comment (mathause) [inline .github/workflows/ci-workflow.yml:15]:
```suggestion
  FORCE_COLOR: 1
```

**Comment by mathause** [comment]
I think it must be not 0. Not sure why keewis went for 3 instead of 1.

- https://force-color.org/
- https://github.com/pydata/xarray/pull/9031
- https://github.com/pytest-dev/pytest/pull/7466

**Pull request merged**